itdagene/core/models.py

```diff
@@ -268,7 +268,7 @@ def current_preference(cls) -> BaseModel:
             pref = Preference.objects.get(active=True)
         except Preference.DoesNotExist:
             year = now().year
-            pref, _ = Preference.objects.get_or_create(
+            pref, __ = Preference.objects.get_or_create(
```

Comment:
I think this might be just a bit better to avoid confusion
```suggestion
            pref, _created = Preference.objects.get_or_create(
```